lua/acf/core/utilities/util_sv.lua

```diff
@@ -135,7 +144,6 @@ do -- Entity saving and restoring
 		end
 	end
 
-	-- Similar to constraint.RemoveAll
```

Comment:
Was this comment inaccurate to the current implementation of ClearConstraints?

Reply:
I believe I added that there when I was first going over things and removed it later. I think git somehow thought it was there before. 

My intention wasn't to document *every* function but mainly just the ones exposed to the user.